wgsl/index.bs

```diff
@@ -4870,8 +4870,8 @@ Consider the following snippet of WGSL:
     }
   </xmp>
 </div>
-The loop body will execute five times.
-Variable `i` will take on values 0, 1, 2, 3, 4, 5, and variable `twice` will take on values 0, 2, 4, 6, 8.
+The loop body will execute six times.
+Variable `i` will take on values 0, 1, 2, 3, 4, 5, and variable `twice` will take on values 0, 2, 4, 6, 8, 10.
```

Comment:
Yes, the loop body executes 6 times, but "Twice" only ever gets to 8.
The i variable is incremented after "twice" is updated.

Reply:
That's right! Which means my correction was in fact also incorrect...